src/pages/detail-page/sections/pc/MainSection.jsx

```diff
@@ -0,0 +1,149 @@
+import { useRef } from 'react';
+import { motion, useScroll, useTransform } from 'motion/react';
+import useElementHeight from '@hooks/useElementHeight';
+import DonationInfo from '@components/DonationInfo';
+import MainTitle from '../../components/MainTitle';
+import Button from '@components/common/Button';
+import DetailTitle from '@pages/detail-page/components/DetailTitle';
+import DetailContent from '@pages/detail-page/components/DetailContent';
+
+const MainSection = () => {
+  // ref를 통해 제목이 브라우저 가장 바닥에 위치한다. (absolute를 사용하면 제목 아래 본문과 이어지지 않는다.)
+  const [titleRef, top] = useElementHeight();
+  const scrollAreaRef = useRef(null);
+
+  // detail title 애니메이션
+  const detailTitleRef = useRef(null);
+  const { scrollYProgress: detailTitleScroll } = useScroll({
+    container: scrollAreaRef,
+    target: detailTitleRef,
+    offset: ['start end', 'start start'],
+  });
+  const detailTitleY = useTransform(
+    detailTitleScroll,
+    [0, 1],
+    ['-1000%', '0%'],
+  );
+  const detailTitleOpacity = useTransform(detailTitleScroll, [0, 0.2], [0, 1]);
+
+  // donation info 애니메이션
+  const donationInfoRef = useRef(null);
+  const { scrollYProgress: donationInfoScroll } = useScroll({
+    container: scrollAreaRef,
+    target: donationInfoRef,
+    offset: ['start end', 'start start'],
+  });
+  const donationInfoY = useTransform(
+    donationInfoScroll,
+    [0, 1],
+    ['-150%', '0%'],
+  );
+  const donationInfoOpacity = useTransform(
+    donationInfoScroll,
+    [0, 0.2],
+    [0, 1],
+  );
+
+  // background 애니메이션
+  const { scrollYProgress: backgroundScroll } = useScroll({
+    container: scrollAreaRef,
+    target: donationInfoRef,
+    offset: ['start end', 'start start'],
+  });
+  const backgroundY = useTransform(backgroundScroll, [0, 1], ['-300%', '0%']);
+  const backgroundOpacity = useTransform(backgroundScroll, [0, 0.2], [0, 1]);
+
+  return (
+    <>
+      {/* 스크롤시 배경 흐리게 */}
+      <motion.div
+        className='absolute inset-0 bg-gradient-to-t from-transparent to-black backdrop-blur'
+        style={{ y: backgroundY, opacity: backgroundOpacity }}
+        ref={donationInfoRef}
+      ></motion.div>
+
+      <div className='flex h-fit w-full justify-center'>
+        <div className='grid h-[calc(100vh-8rem)] w-[95vw] grid-cols-3 grid-rows-4 gap-10'>
+          {/* 제목 + 본문 영역 */}
+          <section
+            className='relative col-start-1 col-end-3 row-start-1 row-end-5 overflow-y-scroll'
+            ref={scrollAreaRef}
+          >
+            <div
+              className='absolute w-[100%]'
+              style={{ top: `calc(100vh - 8rem - ${top}px)` }}
+            >
```

Comment:
`w-[100%]`를 `w-full`로 대체할 수 있습니다.
가독성을 위해 인라인 스타일의 계산식은 따로 변수로 추출해주는 것도 방법일 것 같습니다.

```suggestion
/ 계산 로직을 변수로 추출
            const titleTopPosition = `calc(100vh - 8rem - ${titleHeight}px)`;
            ...
            <div
              className='absolute w-full'
              style={{ top: titleTopPosition }}
            >
```